pl_PL/code-of-conduct.md

```diff
@@ -0,0 +1,72 @@
+* [`performance`](https://github.com/avajs/ava/labels/performance) oferuje pomysły na poprawę wydajności AVA
+
+Etykieta [`help wanted`](https://github.com/avajs/ava/labels/help%20wanted) (prośba o pomoc) i [`good for beginner`](https://github.com/avajs/ava/labels/good%20for%20beginner) (dobre dla początkujących) są szczególnie przydatne.
+
+Możesz znaleźć problem, który jest przypisany lub ma [etykietę `assigned`](https://github.com/avajs/ava/labels/assigned). Sprawdź przed rozpoczęciem tego problemu, ponieważ prawdopodobnie ktoś inny nad nim pracuje.
+
+Chcielibyśmy poprawić [issues `priority`](https://github.com/avajs/ava/labels/priority) (prioritaires) pierwsze. Chcielibyśmy również zobaczyć postęp [issues `low-priority`](https://github.com/avajs/ava/labels/low%20priority) (niski priorytet). [issues `future`](https://github.com/avajs/ava/labels/future) to te, które chcielibyśmy mieć, ale nie od razu. Sprawdź je przed przystąpieniem do pracy, ponieważ w tej chwili możemy nie brać odpowiedzialności za te funkcje.
+
+Jeśli aktualizujesz zależności, upewnij się, że używasz npm@5.6.0 i zobowiązaj się do aktualizacji pliku `package-lock.json`.
+
+### Zobacz na nasz czat
+
+Mamy [chat](https://spectrum.chat/ava). Przyjdź, szpieguj, porozmawiaj z nami i pomóż innym.
+
+## Zgłoś problem
+
+- Issues dotyczą problemów. Skorzystaj z naszego [chat](https://spectrum.chat/ava) lub [Stack Overflow](https://stackoverflow.com/questions/tagged/ava) po wsparcie.
+- Sprawdź błąd w istniejących wyjściach przed otwarciem jednego.
+- Pamiętaj o użyciu najnowszej wersji AVA.
+- Użyj jasnego i opisowego tytułu (w języku angielskim).
+- odaj jak najwięcej informacji: kroki, aby odtworzyć problem, komunikat o błędzie, wersję Node.js, system operacyjny itp. (W języku angielskim).
+- Im więcej czasu poświęcisz na napisanie swojego problemu, tym szybciej go rozwiążemy.
+- [Najlepszym sposobem na odroczenie wyniku jest udowodnienie go nieudanym testem.](https://twitter.com/sindresorhus/status/579306280495357953)
+
+## Prześlij pull request
+
+- Nietrywialne zmiany należy najpierw omówić w kwestii: aby uniknąć wykonywania niepotrzebnej pracy.
+- W przypadku ważnych zadań radzimy jak najszybciej przedstawić swoje prace społeczności w celu uzyskania ich opinii. Otwórz pull request, gdy tylko zrobisz minimum niezbędne, aby zademonstrować swój pomysł. Na tym wczesnym etapie nie martw się, jeśli rzeczy nie są idealne lub w 100% skończone. Dodaj przedrostek [WIP] w tytule i opisz, co jeszcze musisz zrobić. Dzięki temu osoby patrzące na to nie szukają drobiazgów ani nie zgłaszają ulepszeń, które już wiesz, że musisz wprowadzić.
+- Nowym funkcjom muszą towarzyszyć testy i dokumentacja (w języku angielskim).
+- Nie dodawaj zmian niezwiązanych z pull requestem.
+- Sprawdź (Lint) i przetestuj przed wysłaniem pull requestem, uruchamiając `$ npm test`.
+- Złóż pull request [branche](https://github.com/dchelimsky/rspec/wiki/Topic-Branches), nie dla mastera.
```

Comment:
maybe better will be 'wyślij pull request'

Reply:
sure, agree! Thanks!